main.go

```diff
@@ -115,6 +115,9 @@ func mainE(ctx context.Context) error {
 	daemonCommand.PersistentFlags().String(f.Service.Provider.Kind, "", "Provider of the installation. One of aws, azure, kvm.")
 	daemonCommand.PersistentFlags().String(f.Service.Installation.Name, "", "Name of the installation.")
 	daemonCommand.PersistentFlags().String(f.Service.Vault.Host, "", "Host used to connect to Vault.")
+	daemonCommand.PersistentFlags().Bool(f.Service.Security.Letsencrypt.Enabled, false, "Is Lets encrypt enabled in the installation?")
+	daemonCommand.PersistentFlags().Bool(f.Service.Security.RestrictedAccess.Enabled, false, "Is the access to the prometheus restricted to certain subnets?")
+	daemonCommand.PersistentFlags().String(f.Service.Security.RestrictedAccess.Subnets, "", "List of subnets to restrict the access to.")
```

Comment:
Would you mind just even describing in the PR why add these flags?

Reply:
I added them to keep the behavior here: https://github.com/giantswarm/g8s-prometheus/blob/master/helm/g8s-prometheus/templates/prometheus/ingress.yaml

Basically, some installations do not support letsencrypt and some restrict access to prometheus to only some subnets

Reply:
Where do you think I should add something?

Reply:
Nice, I didn't know about that 👍 

Well I try to describe every change in the PR description so it ends up in the git comment to master. This helps to figure out why something was done when going through history.

Reply:
I will rewrite the commit before the merge then :D

Reply:
Done